### PR TITLE
Create filter by institution in events page

### PR DESCRIPTION
**Pull request by Luiz-FS**
**Feature/Bug description:** Create Event Filter Screen by Institution

**Solution:** ...

localhost_8081_events galaxy note ii

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/event/filterEvents/filterEventByInstitution.component.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+(function() {
```
Comment (raonismaneoto) [inline frontend/event/filterEvents/filterEventByInstitution.component.js:1]: test the functions

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: frontend/event/eventController.js [hunk 2]
```diff
@@ -268,6 +275,19 @@
             eventCtrl.toolbarItems = eventCtrl._getToolbarMobileMenuItems();
         };
 
+        eventCtrl.confirmFilter = function confirmFilter() {
+            eventCtrl.events = [];
+            eventCtrl.isFiltering = false;
```
Comment (JuliePessoa) [inline frontend/event/eventController.js]: use the function below

File: frontend/event/eventController.js [hunk 3]
```diff
@@ -268,6 +275,19 @@
             eventCtrl.toolbarItems = eventCtrl._getToolbarMobileMenuItems();
         };
 
+        eventCtrl.confirmFilter = function confirmFilter() {
```
Comment (JuliePessoa) [inline frontend/event/eventController.js:278]: jsdoc

File: frontend/event/eventController.js [hunk 4]
```diff
@@ -268,6 +275,19 @@
...
+            eventCtrl.events = [];
+            eventCtrl.isFiltering = false;
+            eventCtrl._actualPage = 0;
+            eventCtrl._moreEvents = true;
+            eventCtrl.isLoadingEvents = true;
+            return eventCtrl.loadMoreEvents();
+        };
+
+        eventCtrl.cancelFilter = function cancelFilter() {
```
Comment (JuliePessoa) [inline frontend/event/eventController.js:290]: jsdoc

File: frontend/event/events_mobile.html [hunk 5]
```diff
@@ -1,6 +1,7 @@
+
```
Comment (JuliePessoa) [inline frontend/event/events_mobile.html]: remove blank line

File: frontend/event/filterEvents/filter_events_by_institution.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+            </div>
+            <div class="filter-item">
+                <md-checkbox ng-model="filterCtrl.enableAll" ng-change="filterCtrl.enableOrDisableAll()" aria-label="Todos os eventos"></md-checkbox>
+                <span>Todos os eventos</span>
+            </div>
+        </div>
+    </div>
+
+    <div class="filter-buttons-container">
+        <md-button style="color: white; background-color: #7F7F7F;" ng-click="filterCtrl.cancel()">CANCELAR</md-button>
```
Comment (JuliePessoa) [inline frontend/event/filterEvents/filter_events_by_institution.html]: move style

**Review by JuliePessoa — CHANGES_REQUESTED** [review]

**Review by JuliePessoa — APPROVED** [review]

**Review by raonismaneoto — APPROVED** [review]

**Pull request merged**